src/pages/Whattodo.js

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { Table } from "react-bootstrap";
+import { FormattedMessage } from 'react-intl';
+
+export default function Whattodo() {
+  const [Task, setTask] = useState([]);
+
+  const fetchData = async () => {
+    try {
+      const response = await fetch('http://www.boredapi.com/api/activity/');
```

Comment:
nit: move this to `/api/boardApi.js` and move this in a function call `getActivity` to better organize the apis and reusability.